### PR TITLE
Created Todo List with functions and run its tests.

### PR DESCRIPTION
**Pull request by ABDULALI3468**
In this project I did the following things:
I HAVE CHECKED EVERY FUNCTION WITH THE UNIT TEST IN THIS PROJECT.

1) Added the functionality of adding, and removing lists.
2) Added the functionality of updating todos.
3) Added the functionality of Clearing up all checked(completed todos) dynamically.
4) All these things are also stored in local storage dynamically.
5) Added a function to automatically reassign the index values whenever some item is deleted, updated, or added.
6) It is by the ES6 new feature like arrow functions and scopes etc.
7) I have used ### WEB PACK for this project with mode: "development" and it's working as it needs to.

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @ABDULALI3468, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from both members of your programming group. Kindly make sure all partners contribute to the repo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @ABDULALI3468 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

As mentioned by the previous reviewer You have submitted a project, that doesn't have commits from both members of your programming group. Kindly make sure all partners contribute to the repo. And request another code review Good luck 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+
+- npm test or 
+- npm run watch
+## Authors
+
+👤 **Abdul Ali**
+
+- GitHub: [@ABDULALI3468](https://github.com/ABDULALI3468)
+
+## 🤝 Contributing
```
Comment (codepantha) [inline README.md]: - [ ] Please include the details of your coding partner as part of the authors 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUESTED ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights ⭐ 
✔️ Descriptive PR
✔️ Meaningful commit messages
✔️ Linter checks

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wale-prog** [comment]
Oh! thanks for the feedback, I have just included it in my latest commit.

**Review by Lameck1 — APPROVED** [review]
Hi, @ABDULALI3468 & @wale-prog,

Tasks for this project have been successfully implemented 💯.

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**